Production build fails and is slow

Building for production using `yarn build` gives an error:

```
ERROR in ./ammo.js
Module not found: Error: Can't resolve 'fs' in '/home/nazam/Simulator/src'
 @ ./ammo.js 2:1426-1439
 @ ./Sim.tsx
 @ ./components/SimulatorArea.tsx
 @ ./components/Root.tsx
 @ ./index.tsx
```

The build is also really slow (1 hr 20 mins on my machine).

--- a/configs/webpack/common.js
+++ b/configs/webpack/common.js
@@ -5,6 +5,10 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 
 module.exports = {
+  entry: {
+    app: './index.tsx',
+    worker: './worker.ts'
+  },
   resolve: {
     extensions: ['.ts', '.tsx', '.js', '.jsx'],
   },
@@ -58,5 +62,8 @@ module.exports = {
   },
   performance: {
     hints: false,
+  },
+  node: {
+    fs: 'empty',
   },
 };

--- a/configs/webpack/dev.js
+++ b/configs/webpack/dev.js
@@ -7,17 +7,9 @@ const { cwd } = require('process');
 
 module.exports = merge(commonConfig, {
   mode: 'development',
-  entry: {
-    app: './index.tsx',
-    worker: './worker.ts'
-  },
   output: {
     filename: 'js/[name].min.js',
     path: join(cwd(), 'dist'),
   },
   devtool: 'cheap-module-eval-source-map',
-  plugins: [],
-  node: {
-    fs: 'empty',
-  },
 });

--- a/configs/webpack/prod.js
+++ b/configs/webpack/prod.js
@@ -15,7 +15,6 @@ module.exports = merge(commonConfig, {
     path: resolve(__dirname, '../../dist'),
     publicPath: '/',
   },
-  devtool: 'source-map',
   plugins: [],
   node: {
     fs: 'empty',

--- a/configs/webpack/prod.js
+++ b/configs/webpack/prod.js
@@ -17,4 +17,7 @@ module.exports = merge(commonConfig, {
   },
   devtool: 'source-map',
   plugins: [],
+  node: {
+    fs: 'empty',
+  },
 });

--- a/configs/webpack/prod.js
+++ b/configs/webpack/prod.js
@@ -6,17 +6,9 @@ const commonConfig = require('./common');
 
 module.exports = merge(commonConfig, {
   mode: 'production',
-  entry: {
-    app: './index.tsx',
-    worker: './worker.ts'
-  },
   output: {
     filename: 'js/[name].min.js',
     path: resolve(__dirname, '../../dist'),
     publicPath: '/',
-  },
-  plugins: [],
-  node: {
-    fs: 'empty',
   },
 });

